src/stores/game.js

```diff
@@ -731,13 +750,15 @@ export const useGameStore = defineStore('game', {
     async requestStalemate() {
       return new Promise((resolve, reject) => {
         io.socket.get('/game/stalemate', (res, jwres) => {
+          this.consideringOpponentStalemateRequest = false;
```

Comment:
I know why you're setting this in both cases, feels like an easy thing to lose sight of though.